### PR TITLE
Move rgpd-banner outside of content

### LINKED ISSUE #242: Improve zoom-in chip
**Is your feature request related to a problem? Please describe.**
* We show a chip titled `Zoom in to display opened places` since #156.
* Unfortunately the chip is currently hidden by the cookie warning.

**Describe the solution you'd like**
1. Could we show the chip  at the top of the map OR at the center of the map OR above the cookie warning?
2. Could the chip be clickable, setting the zoom level to the minimal required zoom level to show POIs?

**Describe alternatives you've considered**
See above.

**Additional context**
(None.)

### PR DESCRIPTION
**Pull request by caugner**
Fixes #242.

### CODE REVIEW

File: js/main.vue [hunk 1]
```diff
@@ -47,8 +47,8 @@
             {{ $t('zoomtosee') }}
           </v-chip>
         </v-slide-y-reverse-transition>
-        <rgpd-banner />
       </v-content>
+      <rgpd-banner />
     </div>
```
Comment (caugner) [inline js/main.vue:52]: It might need to be one more level up (i.e. after this `</div>`).

**Comment by francois2metz** [comment]
Can you post a screenshot of the fix? Doesn't works for me.

**Comment by caugner** [comment]
> 
> 
> Can you post a screenshot of the fix? Doesn't works for me.

I would love to, but unfortunately `yarn install` fails on Windows.

<details>
<pre>
$ yarn install
yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.2: The platform "win32" is incompatible with this module.
info "fsevents@2.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.12: The platform "win32" is incompatible with this module.
info "fsevents@1.2.12" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > opening_hours@3.5.0" has unmet peer dependency "moment@^2.10.6".
warning " > opening_hours@3.5.0" has unmet peer dependency "country-language@^0.1.7".
warning "vue-linkify > linkifyjs@2.1.9" has unmet peer dependency "jquery@>= 1.11.0".
warning "vue-linkify > linkifyjs@2.1.9" has unmet peer dependency "react@>= 0.14.0".
warning "vue-linkify > linkifyjs@2.1.9" has unmet peer dependency "react-dom@>= 0.14.0".
warning " > vue-mapbox@0.4.1" has incorrect peer dependency "mapbox-gl@^0.53.0".
warning " > vue-jest@4.0.0-beta.2" has unmet peer dependency "jest@^24.x".
warning "vue-jest > ts-jest@23.10.5" has unmet peer dependency "jest@>=22 <24".
[5/5] Building fresh packages...
[1/5] ⠂ mapnik
[2/5] ⠂ ttf2woff2
[3/5] ⠂ core-js
[4/5] ⠂ deasync
[1/5] ⠐ mapnik
[2/5] ⠈ ttf2woff2
[-/5] ⠈ waiting...
[-/5] ⠈ waiting...
error D:\git\github.com\caresteouvert\node_modules\mapnik: Command failed.
Exit code: 1
Command: node-pre-gyp install --fallback-to-build
Arguments:
Directory: D:\git\github.com\caresteouvert\node_modules\mapnik
Output:
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.13.0
node-pre-gyp info using node@12.16.1 | win32 | x64
node-pre-gyp WARN Using request for node-pre-gyp https download
node-pre-gyp info check checked for "D:\git\github.com\caresteouvert\node_modules\mapnik\lib\binding\mapnik.node" (not found)
node-pre-gyp http GET https://mapbox-node-binary.s3.amazonaws.com/mapnik/v4.4.0/node-v72-win32-x64-Release.tar.gz
node-pre-gyp http 403 https://mapbox-node-binary.s3.amazonaws.com/mapnik/v4.4.0/node-v72-win32-x64-Release.tar.gz
node-pre-gyp WARN Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/mapnik/v4.4.0/node-v72-win32-x64-Release.tar.gz
node-pre-gyp WARN Pre-built binaries not found for mapnik@4.4.0 and node@12.16.1 (node-v72 ABI, unknown) (falling back to source compile with node-gyp)
node-pre-gyp http 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/mapnik/v4.4.0/node-v72-win32-x64-Release.tar.gz
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@12.16.1 | win32 | x64
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@12.16.1 | win32 | x64
gyp info spawn C:\Python27\python.EXE
gyp info spawn args [
gyp info spawn args   'D:\\git\\github.com\\caresteouvert\\node_modules\\node-gyp\\gyp\\gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'msvs',
gyp info spawn args   '-G',
gyp info spawn args   'msvs_version=2015',
gyp info spawn args   '-I',
gyp info spawn args   'D:\\git\\github.com\\caresteouvert\\node_modules\\mapnik\\build\\config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'D:\\git\\github.com\\caresteouvert\\node_modules\\mapnik\\common.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'D:\\git\\github.com\\caresteouvert\\node_modules\\node-gyp\\addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\dev\\.node-gyp\\12.16.1\\include\\node\\common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=C:\\Users\\dev\\.node-gyp\\12.16.1',
gyp info spawn args   '-Dnode_gyp_dir=D:\\git\\github.com\\caresteouvert\\node_modules\\node-gyp',
gyp info spawn args   '-Dnode_lib_file=C:\\Users\\dev\\.node-gyp\\12.16.1\\<(target_arch)\\node.lib',
gyp info spawn args   '-Dmodule_root_dir=D:\\git\\github.com\\caresteouvert\\node_modules\\mapnik',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'D:\\git\\github.com\\caresteouvert\\node_modules\\mapnik\\build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
Der Befehl "mapnik-config" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
gyp: Call to 'mapnik-config --ldflags' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (D:\git\github.com\caresteouvert\node_modules\node-gyp\lib\configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:311:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
gyp ERR! System Windows_NT 10.0.18363
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "D:\\git\\github.com\\caresteouvert\\node_modules\\node-gyp\\bin\\node-gyp.js" "configure" "--fallback-to-build" "--
module=D:\\git\\github.com\\caresteouvert\\node_modules\\mapnik\\lib\\binding\\mapnik.node" "--module_name=mapnik" "--module_path=D:\\git\\github.com\\caresteouvert\\node_
modules\\mapnik\\lib\\binding" "--napi_version=5" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v72"
gyp ERR! cwd D:\git\github.com\caresteouvert\node_modules\mapnik
gyp ERR! node -v v12.16.1
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute 'C:\Program Files\nodejs\node.exe D:\git\github.com\caresteouvert\node_modules\node-gyp\bin\node-gyp.js configure --fallba
ck-to-build --module=D:\git\github.com\caresteouvert\node_modules\mapnik\lib\binding\mapnik.node --module_name=mapnik --module_path=D:\git\github.com\caresteouvert\node_mo
dules\mapnik\lib\binding --napi_version=5 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v72' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (D:\git\github.com\caresteouvert\node_modules\node-pre-gyp\lib\util\compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:311:20)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
node-pre-gyp ERR! System Windows_NT 10.0.18363
node-pre-gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "D:\\git\\github.com\\caresteouvert\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp" "install" "--fallback-t
o-build"
node-pre-gyp ERR! cwd D:\git\github.com\caresteouvert\node_modules\mapnik
node-pre-gyp ERR! node -v v12.16.1
node-pre-gyp ERR! node-pre-gyp -v v0.13.0
node-pre-gyp ERR! not ok
</pre>
</details>

**Comment by francois2metz** [comment]
Have you tried to use docker?

**Comment by caugner** [comment]
> Have you tried to use docker?

Had tried with alpine, didn't work (possibly missing the `mapnik` package), will try again:
https://github.com/osmontrouge/caresteouvert/compare/master...caugner:docker

**Comment by caugner** [comment]
`mapnik` is driving me crazy:

```console
$ docker build .
Sending build context to Docker daemon  177.9MB
Step 1/9 : FROM node:12-alpine
 ---> f77abbe89ac1
Step 2/9 : RUN apk add --no-cache git python
 ---> Using cache
 ---> 11ae66e22fca
Step 3/9 : RUN apk add libc6-compat yarn
 ---> Using cache
 ---> 70e475439c3b
Step 4/9 : WORKDIR /git
 ---> Using cache
 ---> 630eb332a24d
Step 5/9 : COPY . .
 ---> Using cache
 ---> fc590fd60515
Step 6/9 : RUN yarn install
 ---> Using cache
 ---> 146e40f01a2e
Step 7/9 : RUN yarn run sprites
 ---> Running in 5886f2680dbc
yarn run v1.22.0
$ spritezero ./static/sprite/caresteouvert ./mapicons && spritezero --retina ./static/sprite/caresteouvert@2x ./mapicons
internal/modules/cjs/loader.js:1208
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: Error loading shared library ld-linux-x86-64.so.2: No such file or directory (needed by /git/node_modules/mapnik/lib/binding/mapnik.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1208:18)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/git/node_modules/mapnik/lib/mapnik.js:31:15)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn run sprites' returned a non-zero code: 1
```

**Pull request closed without merging**